--- /run/user/1000/gvfs/sftp:host=simon-server.local/mnt/dying_ancients/android/lineage_17.1_MartinX3/device/sony/customization/customization.mk	2020-11-25 00:14:15.000000000 +0100
+++ /home/simon/Desktop/customization.mk	2020-11-27 11:53:34.843596978 +0100
@@ -146,12 +146,13 @@
 PRODUCT_COPY_FILES += \
     $(CUST_PATH)/ims/emptyfile:$(TARGET_COPY_OUT_PRODUCT)/priv-app/ims/lib/arm64/bind_mount_lib64_here
 
+# DO NOT ENABLE THIS - IT WILL CREATE A MAJOR SECURITY HOLE (adb is always open, running as root, no auth needed)
 # USB debugging at boot
-PRODUCT_PROPERTY_OVERRIDES += \
-    persist.sys.usb.config=mtp,adb \
-    ro.adb.secure=0 \
-    ro.secure=0 \
-    ro.debuggable=1
+#PRODUCT_PROPERTY_OVERRIDES += \
+#    persist.sys.usb.config=mtp,adb \
+#    ro.adb.secure=0 \
+#    ro.secure=0 \
+#    ro.debuggable=1
 
 # Recovery UI hack for Ganges video mode panels
 ifneq ($(filter ganges, $(SOMC_PLATFORM)),)
