--- /run/user/1000/gvfs/sftp:host=simon-server.local/mnt/dying_ancients/android/ci/device/sony/pdx201/BoardConfig.mk	2020-11-14 23:55:43.000000000 +0100
+++ /home/simon/Desktop/BoardConfig.mk	2020-11-15 22:07:04.822360077 +0100
@@ -50,11 +50,12 @@
     vendor \
     product
 
-# Slightly overprovision dynamic partitions with 50MiB to
-# allow on-device file editing
-BOARD_SYSTEMIMAGE_PARTITION_RESERVED_SIZE := 52428800
-BOARD_VENDORIMAGE_PARTITION_RESERVED_SIZE := 52428800
-BOARD_PRODUCTIMAGE_PARTITION_RESERVED_SIZE := 52428800
+# Overprovision dynamic partitions with 300MiB to
+# allow on-device file editing and OpenGApps, vendor
+# and product remain on 50MiB overprovision
+BOARD_SYSTEMIMAGE_PARTITION_RESERVED_SIZE := 314572800
+BOARD_VENDORIMAGE_PARTITION_RESERVED_SIZE := 52428800
+BOARD_PRODUCTIMAGE_PARTITION_RESERVED_SIZE := 52428800
 
 # Reserve space for data encryption (109553123000-16384)
 BOARD_USERDATAIMAGE_PARTITION_SIZE := 109553106616
