--- /stuff/device/sony/seine/PlatformConfig.mk	2020-11-14 22:29:14.193942686 +0100
+++ /stuff/PlatformConfig.mk	2020-11-14 22:43:03.002359131 +0100
@@ -64,6 +64,8 @@
 BOARD_AVB_VBMETA_SYSTEM_ALGORITHM ?= SHA256_RSA2048
 BOARD_AVB_VBMETA_SYSTEM_ROLLBACK_INDEX := $(PLATFORM_SECURITY_PATCH_TIMESTAMP)
 BOARD_AVB_VBMETA_SYSTEM_ROLLBACK_INDEX_LOCATION := 1
+BOARD_AVB_MAKE_VBMETA_IMAGE_ARGS += --flags 2
+BOARD_AVB_MAKE_VBMETA_SYSTEM_IMAGE_ARGS += --flags 2
 
 # This target has recovery partitions and bootloader
 # will not set the skip_initramfs cmdline bootparam,
